src/main/telemetry/crsf.c

```diff
@@ -267,8 +267,17 @@ int16_t     Roll angle ( rad / 10000 )
 int16_t     Yaw angle ( rad / 10000 )
 */
 
-#define DECIDEGREES_TO_RADIANS10000(angle) ((int16_t)(1000.0f * (angle) * RAD))
+/// reduce angle to +/- 180 degree. Input: angle in 0.1 degree.
```

Comment:
What's the meaning of the triple `/`?

Reply:
It used for explicitly show a description of an entity. In this case, the function. 
It's an old habit from commercial projects. Triple slashes (or //! or //*) are used for doxygen documentation. And some IDEs (like Visual Studio) recognize it as description and show it in hints or something.
I'll remove it if it confuses.